en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+| Flight Task (Library)                                                              |                                      |                                                  |
+| Matrix (Library)                                                                   |                                      |                                                  |
+| Autotuning (Library)                                                               |                                      |                                                  |
+| [Github Actions (Continuous Integration)][Github Actions (Continuous Integration)] |                                      |                                                  |
+| [Jenkins (Continuous Integration)][Jenkins (Continuous Integration)]               |                                      |                                                  |
+| Linux (Toolchain)                                                                  |                                      |                                                  |
+| Mac OS (Toolchain)                                                                 |                                      |                                                  |
+| [Windows (Toolchain)][Windows (Toolchain)]                                         | WSL2                                 |                                                  |
+| [ROS2][ROS2]                                                                       |                                      |                                                  |
+| [Gazebo Classic (Simulation)][Gazebo Classic (Simulation)]                         |                                      |                                                  |
+| [Gazebo (Simulation)][Gazebo (Simulation)]                                         |                                      |                                                  |
+| Flightgear (Simulation)                                                            |                                      |                                                  |
+| JSBSim (Simulation)                                                                |                                      |                                                  |
+| jMAVSim (Simulation)                                                               |                                      |                                                  |
+| SIH (Simulation)                                                                   |                                      |                                                  |
+| Multivehicle Simulation                                                            |                                      |                                                  |
```

Comment:
This is a feature of a simulation type as I understand it, so would be a sub set of gazebo (say).